templates/hoards/thalami/README.md

```
@@ -0,0 +1,47 @@
+# Thalami Hoard
+
+Personal hoard holding per-machine Thalamus files. Each machine you use
+yggdrasil on contributes one `<machine>-thalamus.md` file. Preferences,
+observations, and concerns sync between machines via this repo's git
+history.
+
+## Layout
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)